jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -1088,43 +1096,60 @@ private static final MethodHandle sendResolveMethodHandle(
 		try {
 			MethodHandles.Lookup lookup = new MethodHandles.Lookup(currentClass, false);
 			MethodType type = null;
-			
+			MethodHandle result = null;
+
 			switch(cpRefKind){
 			case 1: /* getField */
-				return lookup.findGetter(referenceClazz, name, resolveFieldHandleHelper(typeDescriptor, loader));
+				result = lookup.findGetter(referenceClazz, name, resolveFieldHandleHelper(typeDescriptor, loader));
```

Comment:
`resolveFieldHandleHelper(typeDescriptor, loader)` will have the same issue as `vmResolveFromMethodDescriptorString` and will also need the accessCheck.

Reply:
Done.  `resolveFieldHandleHelper()` changed to take a `Lookup` object to do the test.
```
	private static final Class<?> resolveFieldHandleHelper(String typeDescriptor, Lookup lookup, ClassLoader loader) throws Throwable {
		MethodType mt = MethodType.vmResolveFromMethodDescriptorString()...
		lookup.accessCheckArgRetTypes(mt);
		return mt.parameterType(0);
```